app/assets/stylesheets/provider/_pop_navigation.scss

```diff
@@ -89,17 +89,33 @@
   }
 
   &-trigger {
-    color: $border-color;
+    color: $pfBlack200;
     cursor: pointer;
     display: block;
-    padding: line-height-times(3 / 4);
     min-width: line-height-times(5 / 2, true);
+    padding: line-height-times(1) line-height-times(.7);
 
     &:hover,
     &.is-toggled,
     &.is-current {
-      background-color: $header-link-background-hover-color;
-      color: $link-color;
+      color: $white;
+    }
+
+    .fa {
+      font-size: line-height-times(.72);
+
+      &.fa-user {
+        background-image: image-url('img_avatar.svg');
+        background-repeat: no-repeat;
+        display: block;
+        margin-top: -.5rem;
+        height: line-height-times(1.6);
+        width: line-height-times(1.6);
+
+        &:before {
+          content: '';
```

Comment:
FYI: I'm adding a bg image in **.fa-user**, then I delete the content in the pseudo element **before**.
It's a different way, compared to the others icons.
This is because we don't have to make any changes in the HTML structure in this PR, or it will break the tests